downstream/modules/platform/con-controller-overview-details.adoc

```diff
@@ -0,0 +1,150 @@
+
+{ControllerNameStart} includes an _Auditor_ type, who can see all aspects of the systems automation, but has no permission to run or change automation, for those that need a system-level auditor. 
+This can also be useful for a service account that scrapes automation information from the REST API. 
+For more information, see Role-Based Access Controls.
+
+= Cloud and autoscaling flexibility
+{ControllerNameStart} features a powerful optional provisioning callback feature that enables nodes to request configuration on demand. 
+This is an ideal solution for a cloud auto-scaling scenario, integrating with provisioning servers like Cobbler, or when dealing with managed systems with unpredictable uptimes. 
+It requires no management software to be installed on remote nodes, the callback solution can be triggered by a call to `curl` or `wget`, and can be embedded in `init` scripts, kickstarts, or preseeds. 
+Access is controlled such that only machines listed in the inventory can request configuration.
+
+= The ideal RESTful API
+The {ControllerName} REST API is the ideal RESTful API for a systems management application, with all resources fully discoverable, paginated, searchable, and well modeled. A styled API browser enables API exploration from the API root at `\http://<server name>/api/`, showing off every resource and relation. Everything that can be done in the user interface can be done in the API - and more.
+
+= Backup and restore
+{PlatformNameShort} can backup and restore your systems or systems, making it easy for you to backup and replicate your instance as required.
+
+= Ansible Galaxy integration
+By including an {Galaxy} `requirements.yml` file in your project directory, {ControllerName} automatically fetches the roles your playbook needs from Galaxy, GitHub, or your local source control. 
+For more information, see {Galaxy} Support.
+
+= Inventory support for OpenStack
+Dynamic inventory support is available for OpenStack. This enables you to easily target any of the virtual machines or images running in your OpenStack cloud.
+
+= Remote command execution
+When you want to perform a simple task on a few hosts, whether adding a single user, updating a single security vulnerability, or restarting a misbehaving service, {ControllerName} includes remote command execution.
+Any task that you can describe as a single Ansible play can be run on a host or group of hosts in your inventory, enabling you to manage your systems quickly and easily. 
+Plus, it is all backed by an RBAC engine and detailed audit logging, removing any questions regarding who has done what to what machines.
+
+= System tracking
+You can collect facts using the fact caching feature. For more information, see Fact Caching.
+
+= Integrated notifications
+{ControllerNameStart} enables you to easily keep track of the status of your automation. 
+
+You can configure:
```

Comment:
Should we use a complete sentence here to introduce the list:

"You can configure the following notifications:", or something similar?